src/components/RichText/RichTextField.tsx

```diff
@@ -50,13 +59,14 @@ export function RichTextField({
   sx,
   className,
   placeholder,
+  showCharacterCount,
   ...props
 }: RichTextFieldProps) {
-  const instanceRef = useRef<EditorCore>();
+  const instanceRef = useRef<EditorJS>();
```

Comment:
EditorCore is a stripped down version meant to be compatible with SSR. The library's implementation of SSR was lacking and so I'm just avoiding it all together. This actual object gives better types in case we want to do something else with the editor later.